core/node/fee_model/src/lib.rs

```diff
@@ -130,8 +143,9 @@ impl BatchFeeModelInputProvider for ApiFeeInputProvider {
     }
 
     /// Returns the fee model parameters.
-    fn get_fee_model_params(&self) -> FeeParams {
-        self.inner.get_fee_model_params()
+    async fn get_fee_model_params(&self) -> anyhow::Result<FeeParams> {
+        self.inner.get_fee_model_params().await
+        // TODO(PE-137): impl APIBaseTokenAdjuster and use it in the ApiFeeInputProvider.
```

Comment:
Do we need to? I think that `get_batch_fee_input_scaled` already covers everything we need (e.g. we either chose current params or the params from the last block, whichever is stricter).

Reply:
We don't. This is a leftover. Removed!